File: eform-client/src/app/common/models/settings/plugins/plugins-settings.model.ts
export class PluginsSettingsModel {
  total: number;
  settingsList: Array<PluginSettingsModel> = [];
}

export class PluginSettingsModel {
  id: number;
  pluginId: number;
  name: string;
  version: string;
  status: number;
  connectionString: string;
}

File: eFormAPI/eFormAPI.Web/Infrastructure/Models/Plugins/PluginsSettingsModel.cs
using System.Collections.Generic;

namespace eFormAPI.Web.Infrastructure.Models.Plugins
{
    public class PluginsSettingsModel
    {
        public int Total { get; set; }

        public List<PluginSettingsModel> SettingsList
            = new List<PluginSettingsModel>();
    }
}